### PR TITLE
fix(dock-manager): add improvements for styling sample

### LINKED ISSUE #118: Dock manager styling sample improvements
The title in the Global Electricity pie chart doesn't fit in the screen. We should make it shorter.

image

Also the sample should define at least one other layout for smaller screen size breakpoint.

image

### PR DESCRIPTION
**Pull request by zhosti**
Closes #118

### CODE REVIEW

**Review by DiyanDimitrov — CHANGES_REQUESTED** [review]
We should dock and pin the floating pane when the user click pin on a floating pane.

File: samples/layouts/dock-manager/styling/src/index.ts [hunk 1]
```diff
@@ -125,16 +160,19 @@ export class DockManagerStylePanes {
...
         this.dockManager.addEventListener("pinPane", () => this.handlePinPane());
+
+        registerIcon("arrow-down", "https://fonts.gstatic.com/s/i/materialiconsoutlined/arrow_drop_down/v15/24px.svg", "material");
+        registerIcon("arrow-up", "https://fonts.gstatic.com/s/i/materialiconsoutlined/arrow_drop_up/v12/24px.svg", "material");
     }
 
     handleClosePane() {
         this.dockManager.activePane.hidden = true;
-        this.dockManager.layout = {...this.layout};
+        this.dockManager.layout = { ...this.layout };
```
Comment (DiyanDimitrov) [inline samples/layouts/dock-manager/styling/src/index.ts:170]: We should use the actual reference of the layout. Same for `handlePinPane`.
```suggestion
        this.dockManager.layout = { ...this.dockManager.layout };
```

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
please remove all these images and email them to me. I will add them to CDN from where we can share these images between SB on all platforms

**Review by DiyanDimitrov — APPROVED** [review]

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
please resolve conflicts

**Review by DiyanDimitrov — APPROVED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**